docs/features/records.md

```diff
@@ -261,8 +370,23 @@ is translated into this code
 ```cs
 public abstract class Person : IEquatable<Person>
 {
-    public abstract string Name { get; }
-    public abstract bool Equals(Person other); // for IEquatable<Person>
+    public string Name { get; }
```

Comment:
Note: Github seems be to rendering as lining up -- the raw markdown seems to be the difference.